cognite_toolkit/cognite_modules/infield/cdf_infield_location/transformations/tr_workorder_oid_workmate_infield_sync_workorders_to_apm_activities.yaml

```diff
@@ -11,6 +11,7 @@ destination:
 ignoreNullFields: true
 shared: true
 action: upsert
+dataSetExternalId: ds_infield_{{default_location}}_app_data
```

Comment:
Here I believe we shouldn't have a data set. If I remember correctly, this is a RAW to data model transformation that does not touch a data set.